plugins/node/opentelemetry-instrumentation-restify/test/utils.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { isAsyncFunction, isPromise } from '../src/utils';
+import * as assert from 'assert';
+
+describe('utils', () => {
+  describe('isPromise', () => {
+    it('should be able to validate a promise to be true', () => {
+      assert.strictEqual(isPromise(Promise.resolve()), true);
```

Comment:
Maybe add a testcase that validates that a non built in Promise (e.g. Bluebird or some home grown variant) is also detected as promise.

Reply:
I've added Bluebird.